mmaction/models/builder.py

```diff
@@ -74,6 +74,10 @@ def build_model(cfg, train_cfg=None, test_cfg=None):
         return build_recognizer(cfg, train_cfg, test_cfg)
     if obj_type in DETECTORS:
         return build_detector(cfg, train_cfg, test_cfg)
+    class_in_mmdet = ['FastRCNN', 'MaxIoUAssignerAVA', 'RandomSampler']
```

Comment:
Does RandomSampler belong to model?

Reply:
emmm, and `MaxIoUAssignerAVA` is also in mmaction2.